week-2/G-conditionals-2/exercise-1.js

```diff
@@ -7,7 +7,9 @@
 */
 
 function negativeOrPositive(number) {
-
+  if (number >= 0) return "postive";
+  else number <= 0;
```

Comment:
This is not quite right, if-else statements have the following shape:

```
if (condition) {
    block of code to be executed if the condition is true
} else { 
    block of code to be executed if the condition is false
}

```
In your code, when the condition is true, the code is correct. But when the condition is false, it tries to execute "number <= 0" which is wrong and uneccessary (you've already checked through your if condition whether it's >=0 and it only goes to the else block if it failed, so no need to check again). It then exits the else block (because there are no curly brackets, then only one statement is within the else block) and only after it's out of the if-else block it returns "negative" (which again is not right, it should return "negative" within the if-else block).

If this doesn't make sense, please let me know and we'll go over it together :)